static/robots.txt

```diff
@@ -1,3 +1,5 @@
 User-agent: *
 Disallow: /experimental/
 Sitemap: https://pub.dev/sitemap.txt
+# TODO: enable this after publishers are launched
+# Sitemap: https://pub.dev/sitemap/publishers.txt
```

Comment:
Can we maybe just make this `https://pub.dev/sitemap-1.txt`  and `https://pub.dev/sitemap-2.txt`.

I mean, this is not an API anyone should be using. It's only for URL discovery. We don't want people using this to guess what publishers/packages exist.